### ISSUE TITLE
Can syncit be used with socket?

### ISSUE THREAD

**Issue by zhu956645239** [Yuyz0112/syncit#16]
Hello, great developer, can syncit follow rrweb to connect with socket? Why must peerjs be used to realize remote control?

**Comment by provinceinnovation** [comment]
Yes! Please see: https://github.com/Yuyz0112/syncit/blob/master/packages/transporter/src/websocket.ts

**Comment by zhu956645239** [comment]
Oh, thank you very much. If you use websockettransporter, can rrweb and syncit share a common connection?

**Comment by Juice10** [comment]
Hi @zhu956645239, thanks for submitting this issue. The syncit project got [moved to the rrweb-io GitHub org](https://github.com/rrweb-io/syncit), and this repo will get archived soon. Would you mind re-submitting your issue there if it is still relevant for the [new 1.0.1 (rrweb 2.0 based) syncit](https://github.com/rrweb-io/syncit)?
